### DISCUSSION TITLE
新人第一帖

### DISCUSSION THREAD

**Discussion by wyqsagiri** [share121/inter-knot#3271] [其他]
我才不会表演那个
就是那个………………………
Ciallo～(∠・ω< )⌒☆

**Comment by ailixiyachuyin** [comment]
这个委托我接了！<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

**Reply by wyqsagiri** [reply]
有绳匠在我就放心了
